### PR TITLE
Fix bug with adding tag `latest`

### LINKED ISSUE #198: The first upload does not contain the latest tag
first upload:npm publish --tag next--verbose
Metadata："dist-tags": { "next": "1.0.0"} , no latest tag

### PR DESCRIPTION
**Pull request by genryxy**
Closes #198 
Fixed bug with adding tag `latest` on first upload in case of existence user tag

### CODE REVIEW

File: src/main/java/com/artipie/npm/Meta.java [hunk 1]
```diff
@@ -44,15 +48,23 @@
      * @return Completion or error signal.
      */
     public Meta updatedMeta(final JsonObject uploaded) {
+        final String latest = "latest";
```
Comment (g4s8) [inline src/main/java/com/artipie/npm/Meta.java:51]: @genryxy it could be a constant

**Review by g4s8 — CHANGES_REQUESTED** [review]
@genryxy just a minor comment

**Comment by genryxy** [comment]
@g4s8 thanks, extracted

**Review by g4s8 — APPROVED** [review]
@genryxy thanks

**Pull request merged**